source/vision/util.py

```diff
@@ -0,0 +1,88 @@
+#vision/util.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2018-2019 NV Access Limited, Babbage B.V.
+
+"""Utility functions for vision enhancement providers.
+"""
+
+from .constants import Role, Context
+import api
+import locationHelper
+
+def getReviewRect():
```

Comment:
This was getReviewRectangle before. I decided to rename these in order to improve consistency.